src/pps/pps.proto

```diff
@@ -36,6 +36,7 @@ message Transform {
   string user = 11;
   string working_dir = 12;
   string dockerfile = 13;
+  bool memory_backed_volume = 14;
```

Comment:
Could we shorten this to `memory_volume`.

Reply:
Sure, can do.